src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetailTabs/Manifest.component.tsx

```diff
@@ -81,7 +80,6 @@ const ManifestComponent = ({
         if (isDeleted) {
             return
         }
-        toggleManagedFields(false)
```

Comment:
is it intentional?

Reply:
@AbhishekA1509 had pointed out an issue with this. But really need to verify with product team before doing anything here. So reverting for new.